extensions/sql-database-projects/src/dialogs/addDatabaseReferenceDialog.ts

```diff
@@ -274,7 +276,6 @@ export class AddDatabaseReferenceDialog {
 		this.formBuilder!.insertFormItem(<azdata.FormComponent>this.dacpacFormComponent, 2);
 
 		this.locationDropdown!.values = constants.locationDropdownValues;
-		this.locationDropdown!.value = constants.differentDbSameServer;
```

Comment:
when the dacpac radio button is clicked, the location should stay the same as whatever the previously selected value was.